pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/Utils.groovy

```diff
@@ -610,6 +627,19 @@ class Utils {
 
         BulkChange bc = new BulkChange(j)
         try {
+            // Check if QuietPeriod option is specified
+            QuietPeriod quietPeriod = (QuietPeriod) rawOptions.find { it instanceof QuietPeriod }
+            if (quietPeriod != null) {
+                j.setQuietPeriod(quietPeriod.quietPeriod)
+            } else {
+                String quietPeriodName = Jenkins.getActiveInstance().getDescriptorByType(QuietPeriod.DescriptorImpl.class)?.getName()
+                // If the quiet period was set by the previous build, wipe it out.
+                System.err.println("options: ${previousOptions}")
```

Comment:
Is this visible to users, or is it just for debugging? (Just curious)

Reply:
That...is me missing a debug println.